src/main/java/com/entropyteam/entropay/employees/clients/models/Client.java

```diff
@@ -0,0 +1,61 @@
+package com.entropyteam.entropay.employees.clients.models;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "clients", schema = "employees")
```

Comment:
I'd keep the schema in the connection properties.

Reply:
@agustinb4 If we are not planning to use another schema in the same service, I agree. Wdyt?

Reply:
@arael82 Yes, totally agreed

Reply:
Ok, done